docs/learn/learn-nft.md

```diff
@@ -1,14 +1,14 @@
 ---
 id: learn-nft
 title: NFTs
-sidebar_label: NFTs
+sidebar_label: Introduction to NFTs
 description: An explanation of the NFT landscape in the Polkadot ecosystem.
 keywords: [NFT, non-fungible token, NFT 2.0]
 slug: ../learn-nft
 ---
```

Comment:
Drücken Sie lange auf ein Element in der Zwischenablage, um es anzupinnen. Nicht angepinnte Elemente werden nach einer Stunde gelöscht.